configuration.nix

```diff
@@ -59,22 +55,23 @@ inputs: let
         enableExtraDiagnostics = true;
 
         nix.enable = true;
+        markdown.enable = true;
+
         html.enable = isMaximal;
         css.enable = isMaximal;
         sql.enable = isMaximal;
         java.enable = isMaximal;
         ts.enable = isMaximal;
         svelte.enable = isMaximal;
         go.enable = isMaximal;
+        elixir.enable = isMaximal;
         zig.enable = isMaximal;
         python.enable = isMaximal;
         dart.enable = isMaximal;
-        elixir.enable = isMaximal;
         bash.enable = isMaximal;
-        terraform.enable = isMaximal;
-        nim.enable = false;
         tailwind.enable = isMaximal;
         typst.enable = isMaximal;
+        nim.enable = isMaximal;
```

Comment:
Nim module doesn't work on Darwin https://github.com/PMunch/nimlsp/issues/178#issue-2128106096. The issue hasn't been resolved yet.

Reply:
right, I forgot about this entirely. I'll change this to `!isDarwin`